packages/@aws-cdk/aws-cloudformation/test/integ.core-deps.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Real replaceDependency use case to test
+ *
+ * TestStack verification steps:
+ * - Deploy with `--no-clean`
+ * - Verify that the CloudFormation stack LogRetention CfnResource dependencies list CustomPolicy, not DefaultPolicy
+ *
+ * TestNestedStack verification steps:
+ * - Deploy with `--no-clean`
+ * - Verify that Stack2 lists Stack1 in DependsOn
+ */
```

Comment:
leftover?

Reply:
Not strictly leftover - I was following [this test](https://github.com/aws/aws-cdk/pull/20419#issuecomment-1319277855) as an example, which spelled out exactly what they were looking at in the template to verify it was desireable.